Tabloid/Repositories/CommentRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Tabloid.Utils;
+using Tabloid.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Data.SqlClient;
+
+namespace Tabloid.Repositories
+{
+    public class CommentRepository : BaseRepository, ICommentRepository
+    {
+        public CommentRepository(IConfiguration configuration) : base(configuration) { }
+
+        public List<Comment> GetCommentsByPostId(int id)
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"
+                SELECT c.Id, c.UserProfileId, Subject, c.Content, c.CreateDateTime, u.FirstName, u.LastName, u.DisplayName, 
+                    u.Email, 
+                    u.CreateDateTime AS UserProfileDateCreated,
+                    u.ImageLocation AS AvatarImage,
+                    u.UserTypeId 
+                  FROM Comment c
+                  JOIN Post p on c.PostId = p.Id
+                  Join UserProfile u on p.UserProfileId = u.Id
+              WHERE PostId = @Id";
+
```

Comment:
Here you will want to add an order by statement. According to the ticket, the comments should be ordered by creation date with the most recent at the top.

Reply:
Checked off!